public/pages/Main/Main.tsx

```diff
@@ -156,6 +159,15 @@ export default class Main extends Component<MainProps, MainState> {
     }
   };
 
+  dataSourceFilterFn = (dataSource: SavedObject<DataSourceAttributes>) => {
+    const engineVersion = dataSource?.attributes?.dataSourceVersion || "";
```

Comment:
let's call this `dataSourceVersion` so we stay consistent in naming, similarly below let's call that `installedPlugins`

Reply:
addressed